packages/core/upload/admin/src/hooks/useFolders.ts

```diff
@@ -33,7 +40,7 @@ export const useFolders = ({ enabled = true, query = {} } = {}) => {
       },
       filters: {
         $and: [
-          ...(paramsExceptFolderAndQ?.filters?.$and ?? []),
+          // ...(paramsExceptFolderAndQ?.filters?.$and ?? []),
```

Comment:
why is this commented? should it be deleted entirely?

Reply:
no it is just a comment that remains during the implementation, I am gonna remove it

Reply:
and add again the rest instruction